pyproject.toml

```diff
@@ -1,10 +1,89 @@
 [build-system]
 requires = [
-  "setuptools",
-  "setuptools_scm",
-  "wheel",
+  "setuptools>=62.1",
+  "setuptools_scm[toml]>=6.2",
+  "wheel",]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ndcube"
+description = "A package for multi-dimensional contiguous and non-contiguous coordinate aware arrays."
+readme = "README.rst"
+requires-python = ">=3.9"
+license = { file = "licenses/LICENSE.rst", content-type = "text/plain" }
+authors = [
+  { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
+]
+dependencies = []
+dynamic = ["version"]
+
+[project.optional-dependencies]
+tests = [
+    "pytest",
+    "pytest-doctestplus",
+    "pytest-cov",
+]
+docs = [
+    "sphinx",
+    "sphinx-automodapi",
+    "tomli; python_version <\"3.11\"",
+]
+
+[project.urls]
+repository = "https://docs.sunpy.org/projects/ndcube/"
```

Comment:
We should expand these with docs and repository and homepage?

Reply:
https://github.com/sunpy/package-template/issues/34